observer/observer.go

```diff
@@ -12,12 +12,16 @@ import (
 type Observer struct {
 	logger   blog.Logger
 	monitors []*monitor
+	shutdown func()
 }
 
-// Start spins off a goroutine for each monitor and then runs forever.
+// Start spins off a goroutine for each monitor.
 func (o Observer) Start() {
 	for _, mon := range o.monitors {
 		go mon.start(o.logger)
 	}
-	select {}
+}
```

Comment:
See paired comment on cmd/boulder-observer/main.go:
```suggestion
	select {}
}
```